PlaidLink.js

```diff
@@ -114,6 +114,12 @@ PlaidLink.propTypes = {
 
   // Optional props
 
+  // You can configure Link to return only the accounts that are a
+  // checking or savings account subtype
+  // see https://plaid.com/docs/#auth-filtering-institutions-in-link
+  // and https://plaid.com/docs/#filtering-institutions-in-link
+  accountSubtypes: PropTypes.object,
```

Comment:
Consider using `PropTypes.shape` to specify the expected shape of the object. 

```suggestion
  accountSubtypes: PropTypes.shape({
    credit: PropTypes.arrayOf(PropTypes.string),
    depository: PropTypes.arrayOf(PropTypes.string),
    loan: PropTypes.arrayOf(PropTypes.string),
  }),
```